shared/src/main/scala/carldata/series/MathModule.scala

```diff
@@ -0,0 +1,13 @@
+package carldata.series
+
+object MathModule {
```

Comment:
We support scala.js but this is new feature so lets implement it in separate issue.